keel-core/keel-core.gradle.kts

```diff
@@ -35,6 +34,7 @@ dependencies {
   api("net.swiftzer.semver:semver:1.1.0")
   api("org.jetbrains.kotlinx:kotlinx-coroutines-core")
   api("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j")
```

Comment:
Why did you add this commons lang dependency? I don't see it used anywhere, unless I am missing something

Reply:
It was coming in with `kork-artifacts`. If you search for `import org.apache.commons.lang3` you'll find lots of usages.

Reply:
ah, ok, got it! so I was missing something :D